stream_alert/rule_processor/handler.py

```diff
@@ -24,6 +27,10 @@
 from stream_alert.rule_processor.rules_engine import StreamRules
 from stream_alert.rule_processor.sink import StreamSink
 from stream_alert.shared.metrics import Metrics
+from stream_alert.shared.stats import time_me
+
+NUM_WORKERS = 6
```

Comment:
It looks like this returns 2 cpus in a lambda configured with 512 mb ram.. may be different for larger ram allocations.